.github/workflows/windows-npm.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Tests on Windows: `nvm install`'
+
+on: [pull_request, push]
+
+jobs:
+  node:
+    name: 'MSYS fail prefix nvm install'
+    runs-on: windows-latest
+    steps:
+      - name: Retrieve nvm
+        shell: bash
+        run: |
+          mkdir vp "$HOME/.nvm"
+          curl -sSLo "$HOME/.nvm/nvm.sh" "https://raw.githubusercontent.com/${GITHUB_REPOSITORY}/${GITHUB_SHA}/nvm.sh"
+          source "$HOME/.nvm/nvm.sh"
+          ! nvm install --lts
```

Comment:
Because it does not work by default because it is using an NPM_PREFIX. It was even causing a infinite loop problem. So I added this test to ensure that it does not work, but at least it ends up not in an infinite loop.

Reply:
What is using an NPM_PREFIX and why?

Reply:
I have no idea, that is how the system is set up on github action using windows-latest...

Reply:
yes that is what I do for the other tests that are passing, but I felt like keeping this test to know that by default it does not work.
Do you want me to remove this test then ?